tools/picard/picard_AddOrReplaceReadGroups.xml

```diff
@@ -1,4 +1,4 @@
-<tool name="AddOrReplaceReadGroups" id="picard_AddOrReplaceReadGroups" version="1.126.0-gx-1">
+<tool name="AddOrReplaceReadGroups" id="picard_AddOrReplaceReadGroups" version="1.135-gx-1">
```

Comment:
I think we can safely abandon the ```-gx-1``` part. It is a relict of when we were not sure how to version.